cypress.json

```
@@ -0,0 +1 @@
+{ "baseUrl": "http://localhost:3000" }
```

Should we add a configuration for `testFiles` here so we can enforce the naming for test files according to our [Cypress convention](https://nimblehq.co/compass/development/code-conventions/javascript/cypress/#naming)?

```suggestion
{
  "baseUrl": "http://localhost:3000",
  "testFiles": ["**/*.spec.ts"]
}
```

---

done 👍